File: Computing1/Past_Computing1/22-1_컴기_Homework/Homework/HW20_v3/P2.c
#include <stdio.h> 
#include <stdlib.h>
#include <string.h>

int main(int argc, char *argv[]){
    char *input_filename = argv[1]; // name of input file
    // YOUR CODE HERE

    FILE *infile;
    int size;
    int count; 
    
    infile = fopen(input_filename, "r");

    if (infile == NULL) {return 0;}
    else {
        fseek(infile, 0, SEEK_END); // 파일의 끝으로 이동
        size = ftell(infile); // 파일의 현재 위치 
        fseek(infile, 0, SEEK_SET); // 파일의 맨 앞으로 이동 

        int arr[size];

        while (fscanf(infile, "%d", arr) != EOF){
            printf("%d", arr[0]);
        }
    }
    fclose(infile);

    return 0; 
}

File: Computing1/Past_Computing1/22-1_컴기_Homework/Homework/HW3/P2.py
def P2(L1: list, L2: list, L3: list) -> int:
    ans = 0
    ### Modify code here ###
    
    # 각 리스트의 길이를 계산, 길이들 중 가장 긴 길이를 선택
    ans += max(len(L1), len(L2), len(L3))

    ### End of your code ###
    return ans